.github/workflows/build-test-windows.yml

```diff
@@ -20,7 +20,10 @@ concurrency:
 
 jobs:
   windows-unittest:
-    runs-on: windows-2019
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ windows-2019 ]
```

Comment:
If we're on the same page, this is going to include `windows-latest` in a separate PR that addresses a different Windows 2022, bug, right?

Reply:
Correct.